object_manager.go

```diff
@@ -304,6 +305,40 @@ func (objMgr *ObjectManager) GetFixedAddress(netview string, cidr string, ipAddr
 	return &res[0], nil
 }
 
+func (objMgr *ObjectManager) UpdateFixedAddress(netview string, cidr string, ipAddr string, macAddress string, vmID string) (*FixedAddress, error) {
```

Comment:
Thanks!

NOTE that, the point is, if the method is designed to update very specific field(s), the name of the method should indicate so. Example is UpdateNetworkViewEA: https://github.com/infobloxopen/infoblox-go-client/blob/master/object_manager.go#L133

Oh, BTW, it looks like UpdateNetworkViewEA is not in the interface definition (https://github.com/infobloxopen/infoblox-go-client/blob/master/object_manager.go#L9) Maybe it should be included?

Thanks again!